starters/next-react-query-tailwind/package.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Not sure why this is showing as a completely new file - the "deleted" version is below. I updated the name and description to remove the 12, and removed the ^ from all of the dependencies to lock their versions.